### ISSUE TITLE
allow keyboard navigation between inline actions

### ISSUE THREAD

**Issue by akosyakov** [eclipse-theia/theia#8262]
For instance in toolbars and plugin tree views.

It should be possible to navigate with tab and arrows, and execute with enter.
